.github/workflows/mesh.b2b.deploy.link.yml

```diff
@@ -1,7 +1,6 @@
-name: B2B Web SDK - Test & Deploy
+name: B2B Web SDK - Test & Deploy Link
 on:
-  release:
-    types: [published]
+  workflow_dispatch:
```

Comment:
I think you don't need the `:` in this case: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch

Reply:
fixed